gemelli/rpca.py

```diff
@@ -39,6 +42,11 @@ def phylogenetic_rpca(table: biom.Table,
        gemelli.
     """
 
+    # loads NewickFormat tree as bp.BP tree; loads & parses tree faster
+    phylogeny = get_bp(phylogeny)
```

Comment:
Can you copy the get_bp code instead of importing the function? 
```suggestion
    with open(str(newickfmt)) as treefile:
        # The file will still be closed even though we return from within the
        # with block: see https://stackoverflow.com/a/9885287/10730311.
        phylogeny = parse_newick(treefile.readline())

```